lib/Localization/app_localization_delegate.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/widgets.dart';
+import 'package:marvelapp_flutter/Localization/app_localizations.dart';
```

Comment:
```suggestion
import 'package:marvelapp_flutter/presentation/localization/app_localizations.dart';
```

Reply:
localization is a part of presentation layer

Reply:
move it to presentation